style.css

```diff
@@ -408,14 +423,213 @@ section ul .linked-in,
+    gap: 20px;
+  }
+
+  .nav-link {
+    color: #000;
+    font-size: 15px;
+  }
+
+  /* -- -------------------Styling the Home Page--------------------- -- */
+
+  .main-page {
+    background: url('images/desktop-homeBG.jpeg');
+    padding: 140px 260px;
+  }
+
+  .home-page-details {
+    margin: 0;
+  }
+
+  h1 {
+    padding: 0;
+  }
+
+  h1,
+  .about-myself-heading {
+    font-size: 48px;
+    line-height: 60px;
+    color: #091e42;
+  }
+
+  .about-me-text {
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+  }
+
+  .social-links-container {
+    gap: 20px;
+  }
+
+  .social-links {
+    padding-bottom: 0;
+  }
+
+  .social-links h2 {
+    margin-bottom: 30px;
+  }
+
+  .home-margin-bottom {
+    margin-bottom: 12px;
+  }
+
+  /* -- -------------------Styling the Works Page--------------------- -- */
+  .work-section {
+    gap: 142px;
+  }
+
+  .grid-item-1 {
+    margin-top: 140px;
+  }
+
+  .grid-item {
+    margin: 0 142px;
+  }
+
+  .card-container {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 22px;
+    padding: 24px;
+  }
+
+  .desktop-work-img {
+    display: block;
+  }
+
+  .mobile-work-img {
+    display: none;
+  }
+
+  .reorder-img {
+    order: 1;
+  }
+
+  .line-break {
+    display: none;
+  }
+
+  .work-title {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .h3-common-styles {
+    font-size: 18px;
+    line-height: 24px;
+  }
+
+  .work-role,
+  .work-year {
+    color: #6b778c;
+    font-weight: 400;
+    font-size: 18px;
+  }
+
+  .work-description {
+    font-size: 16px;
+    margin-bottom: 12px;
+  }
+
+  .works-btns {
+    margin-bottom: 28px;
+  }
+
+  .see-project-btn {
+    background: var(--color-white);
+  }
+
+  /* -- -------------------Styling the About Myself Page--------------------- -- */
+  .about-myself-card {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 25px;
+    padding: 142px;
+  }
+
+  .about-myself-details {
+    padding-left: 0;
+  }
+
+  .about-myself-heading {
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  .about-myself-text {
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .about-myself-text,
+  .social-links-heading,
+  .social-links-container {
+    margin-bottom: 18px;
+  }
+
+  .language-list {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    gap: 12px;
+  }
+
+  .list-item-icon {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .list-item {
+    margin-bottom: 0;
+    padding: 18px;
+  }
+
+  .list-item,
+  .list-item-icon {
+    margin-bottom: 12px;
+  }
+
+  .languages-container {
+    padding-bottom: 0;
+  }
+
+  /* -- -------------------Styling the Contact Page--------------------- -- */
+  .contact-page {
+    background-image: url(images/desktop-contact.png);
+  }
+
+  .contact-flex-container {
+    padding: 142px 320px 127px 320px;
+  }
+
+  .contact-form li input,
+  .contact-form li textarea {
+    margin-bottom: 27px;
```

Comment:
- [ ] :thinking: Kindly make sure your contact form matches the Figma design as recommended in the project requirements by adjusting the width.

Your design:
![Screenshot from 2023-01-19 14-45-31](https://user-images.githubusercontent.com/97834160/213448196-cc8b8ff5-48f0-47c6-b7be-5c397856f4ff.png)
Figma design:

![Screenshot from 2023-01-19 14-43-28](https://user-images.githubusercontent.com/97834160/213448278-acec44c1-e35b-4989-9849-b5ce75813ec1.png)